__init__.py

```diff
@@ -376,21 +387,29 @@ def get_harmonic_thermo(self, hnu):
         ZPE_hmodes = []  # Zero point energy for mode
         Z_hmodes = []  # partition function for mode
         e_excitation_hmodes = []  # principle energy of mode
+        entropic_energy_hmodes = [] #entropy contribution per mode
+        x = 0.
 
         for e in hnu:
             if e.imag == 0 and e.real >= 0.010:
                 Z_mode = 1./(1.-np.exp(-e.real/(self.kT)))
                 ZPE = 0.5 * e.real
                 e_min_exitation = e.real
+                x = e_min_exitation/self.kT
+                entropic_energy_hmodes.append(self.kT*(x/(np.exp(x)-1.) - np.log(1.-np.exp(-x))))
```

Comment:
Lines should be maximum 79 characters long. 
See https://www.python.org/dev/peps/pep-0008/#maximum-line-length